icu4c/source/common/unicode/stringpiece.h

```diff
@@ -176,6 +177,16 @@ class U_COMMON_API StringPiece : public UMemory {
    */
   StringPiece(const StringPiece& x, int32_t pos, int32_t len);
 
+#ifndef U_HIDE_INTERNAL_API
+  /**
+   * Converts to an std::string_view().
```

Comment:
Nit: I prefer "a" not "an" -- assuming "std" is pronounced "standard" not "es tee dee"... and now I see that in PR #3071 you also wrote "a std::string_view" :-)  (uhash.h)

Reply:
Done.